apps/ui/src/composables/useSpaces.ts

```diff
@@ -15,6 +13,12 @@ export function useSpaces() {
   const loading = ref(false);
   const loadingMore = ref(false);
   const loaded = ref(false);
+  const protocol = ref('snapshot' as ExplorePageProtocol);
+
+  watch(protocol, toFilter => {
+    handleProtocolChange(toFilter);
+  });
+
```

Comment:
We tend to sort effects, so this would be at the bottom.

Reply:
Interesting, but here it is in middle https://github.com/snapshot-labs/sx-monorepo/blob/master/apps/ui/src/composables/useMarkdownEditor.ts#L27 🙈 will change in `useSpaces` for now